crates/services/txpool/src/txpool/tests.rs

```diff
@@ -42,8 +50,8 @@ use std::{
 
 const GAS_LIMIT: Word = 1000;
 
-#[tokio::test]
-async fn insert_simple_tx_succeeds() {
+#[test]
+fn insert_simple_tx_succeeds() {
```

Comment:
Nice, we can remove it because we did the refactoring in the `TxPool`. Previouslt `insert_inner` was `async`=)